Cargo.lock

```diff
@@ -1563,6 +1567,14 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "common-sql"
+version = "0.1.0"
+dependencies = [
+ "common-exception",
+ "sqlparser",
+]
+
```

Comment:
It seems this crate only involves `expr_visitor`, what about rename it to `common-ast`? Since the name `common-sql` looks too common.

Reply:
I don't have a strong opinion about the naming, I will rename it in other MR.